src/main/java/foxtrot/uniform/charlie/kilo/model/dto/UserRegistrationDto.java

```diff
@@ -0,0 +1,45 @@
+package foxtrot.uniform.charlie.kilo.model.dto;
+
+import foxtrot.uniform.charlie.kilo.annotations.ValidateEmail;
+import foxtrot.uniform.charlie.kilo.annotations.ValidatePassword;
+import javax.validation.constraints.Size;
+
+@ValidatePassword(password = "password", repeatPassword = "repeatPassword",
+        message = "Passwords do not match!")
+public class UserRegistrationDto {
+    @ValidateEmail
+    private String email;
+    @Size(min = 5, max = 16)
```

Comment:
You have your custom annotation for password validation, keep all validation logic in one place